generator/templates/rust/crate/Cargo.toml.mustache

```diff
@@ -28,6 +28,9 @@ license.workspace    = true
 repository.workspace = true
 keywords.workspace   = true
 categories.workspace = true
+{{#NotForPublication}}
+publish              = ["not-for-publication"]
```

Comment:
nit: maybe simply false here: https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish-field

Reply:
D'oh! I should keep reading next time.